content/en/docs/setup/production-environment/tools/kubeadm/setup-ha-etcd-with-kubeadm.md

```diff
@@ -273,22 +273,21 @@ on Kubernetes dual-stack support see [Dual-stack support with kubeadm](/docs/set
 
 1. Optional: Check the cluster health.
 
-   ```sh
-   docker run --rm -it \
-   --net host \
-   -v /etc/kubernetes:/etc/kubernetes registry.k8s.io/etcd:${ETCD_TAG} etcdctl \
-   --cert /etc/kubernetes/pki/etcd/peer.crt \
-   --key /etc/kubernetes/pki/etcd/peer.key \
-   --cacert /etc/kubernetes/pki/etcd/ca.crt \
-   --endpoints https://${HOST0}:2379 endpoint health --cluster
-   ...
-   https://[HOST0 IP]:2379 is healthy: successfully committed proposal: took = 16.283339ms
-   https://[HOST1 IP]:2379 is healthy: successfully committed proposal: took = 19.44402ms
-   https://[HOST2 IP]:2379 is healthy: successfully committed proposal: took = 35.926451ms
-   ```
+    ```sh
+    ETCDCTL_API=3 etcdctl \
```

Comment:
```suggestion
    # If etcdctl isn't available, you can run this tool inside a container image.
    # You would do that directly with your container runtime using a tool such as
    # crictl run
    # - and not through Kubernetes.
    ETCDCTL_API=3 etcdctl \
```

Let's explain to people where `etcdctl` comes from.

Reply:
added the note